components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -320,6 +320,7 @@ struct BlockchainToken {
   bool visible = true;
   string token_id = "";  // Non-empty for ERC721 tokens.
   string coingecko_id = "";
+  string? chain_id = "";
```

Comment:
I am confused about why this is defined as optional? Seems like we're initialize it to empty string anyway.

Reply:
Updated to `string chain_id;`